src/main/scala/scorex/core/api/http/ActorHelper.scala

```diff
@@ -0,0 +1,15 @@
+package scorex.core.api.http
+
+import akka.pattern.ask
+import akka.actor.ActorRef
+import akka.util.Timeout
+
+import scala.concurrent.Future
+import scala.reflect.ClassTag
+
+trait ActorHelper {
```

Comment:
This trait is not specific to this package and could be useful elsewhere in scorex. I would suggest moving it to `scorex.core.utils`.

Reply:
I think so too. So I will work according your comments and prepare pr in the nearest future.